battleship.py

```diff
@@ -95,8 +95,13 @@ def createShip():
 Returns: bool
 '''
 def checkShip(grid, ship):
-    return
-
+    checkShip = False
+    for i in range(len(ship)):
+        if grid[ship[i][0]][ship[i][1]] == EMPTY_UNCLICKED:
+            checkShip = True
+        else:
+            return False
+    return checkShip

```

Comment:
@Vedvundmsit When does the 104th line will execute? We can discuss this here or in person too.